data/campaigns/Winds_of_Fate/scenarios/09_Heart_Mountains.cfg

```diff
@@ -0,0 +1,651 @@
+            [/objective]
+            [objective]
+                description= _ "Death of Gorlack"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Arinexis"
+                condition=lose
+            [/objective]
+            [objective]
+                description= _ "Death of Zedrix"
+                condition=lose
+            [/objective]
+            {TURNS_RUN_OUT}
+            [gold_carryover]
+                bonus=yes
+                carryover_percentage=20
+            [/gold_carryover]
+            [note]
+                description= _ "A winter time schedule is in effect."
+            [/note]
+        [/objectives]
+    [/event]
+
+    [event]
+        name=start
+        # Initial scattering of wildlife.
+        # Small and herbivorous creatures.
+        {SCATTER_UNITS 1 "Frost Stoat" 8 (
+            {EVERYWHERE}
+            terrain=A*
+            [not]
+                terrain=*^V*
+            [/not]
+        ) (side=2)}
+        {SCATTER_UNITS 4 "Great Icemonax" 6 (
+            {EVERYWHERE}
+            terrain=Ha
+        ) (side=2)}
+        {SCATTER_UNITS 1 "Gryphon" 12 (
+            {EVERYWHERE}
+            [not]
+                terrain=*^V*,Ke,Ce
+            [/not]
+        ) (side=2)}
+        # Wolves
+        {SCATTER_UNITS 6 "Direwolf" 6 (
+            {EVERYWHERE}
+            terrain=*^F*
+        ) (side=3)}
+        # Bears
+        {SCATTER_UNITS 4 "Cave Bear" 12 (
+            {EVERYWHERE}
+            terrain=M*
+            [not]
+                terrain=Mv
+            [/not]
+        ) (side=4)}
+        # Yeti
+        {SCATTER_UNITS 2 "Yeti" 12 (
+            {EVERYWHERE}
+            terrain=Ms
+        ) (side=5)}
+        [remove_shroud]
+            terrain=*^Ebn
+            radius=2
+        [/remove_shroud]
+        [delay]
+            time=1000
+        [/delay]
+        {NAMED_UNIT 6 (Drake Glider) 14 38 Klanir _"Klanir" ()}
+        [scroll_to_unit]
+            id=Klanir
+        [/scroll_to_unit]
+        [delay]
+            time=1000
+        [/delay]
+        [move_unit]
+            id=Klanir
+            to_x= 6, 5
+            to_y=35,36
+        [/move_unit]
+        [scroll_to_unit]
+            id=Gorlack
+        [/scroll_to_unit]
+        [message]
+            speaker=Klanir
+            image_pos=right
+            mirror=yes
+            message= _ "Dominant Gorlack, I bring ill tidings of the Hunt.
+
+The Flight of Karron was decimated by a pack of humans upon the grassy stretches.
+The last word from Dominant Karron spoke of their striving for a retreat to your eyrie.
+
+In a noble attempt to aid the imperiled Flight of Karron...
+My Dominant Vladnir fell to our prey.
+
+His flight was scattered east, deep into the greatland."
+        [/message]
+        [message]
+            speaker=Gorlack
+            message= _ "<span color='grey' >..................</span>"
```

Comment:
Every time Gorlack speaks in this scenario his text is rendered in grey. I can only assume this is a deliberate intention to convey his grief from the previous scenario, but during the game I thought it was a bug. Not sure if I can think of a better way to convey this emotion, however.

Reply:
Yeah, I will just get rid of it.

Reply:
removed it